### ISSUE TITLE
Add custom property of traits that have not been registered, yet

### ISSUE THREAD

**Issue by jceb** [decentralized-identity/did-traits#12]
In the meeting from 2024-11-12 the idea was presented to support trait definitions that have not been registered, yet. The proposal received some push-back:

- DID method authors who identify an important new trait are inherently incentivized to register this trait in this repository. The definition of a trait with property name, title and description is already 90% of the PR that is required to register the new trait.
- Traits could linger in the custom section forever and create their own "custom" standard.
- Adding additional properties to the traits document is already supported since the defined JSON schema doesn't make additional properties illegal.

Example:

```json
{
  "name": "mymethod",
  "custom": [
    {
       "mynewtrait": {
          "value": true,
          "title": "My new trait",
          "description": "This is a super important new trait"
       }
    } 
  ]
}
```

We will discuss this feature at the upcoming meeting on 2024-11-26.

**Comment by ottomorac** [comment]
Yes. Not a bad idea to be suggestion of new traits. But the effort to curate the "standard" lies in the working group itself. I would say we for now should restrict suggestion on new traits to the issues section of the github repo as we have been doing.
